### ISSUE TITLE
In the "LinalgFusionOfTensorOps" node fusion pass linalg.conv Operator does not seems to be inside the Generic Operator Region unlike other pointwise operators add/mul .. 

### ISSUE THREAD

**Issue by dinkdeep** [tensorflow/tensorflow#48363]
Looks like the linalg.conv operator  are not placed inside the Generic Operator Region
 For our custom requirement we would like to use and leverage on the "Node fusion capability of Regions" offered by the "LinalgFusionOfTensorOps"  , The pass  seems to work point wise operators but not for the  operators like linalg.conv/MaxPool/BatchNorm/AVvgPool etc  
 
We have our own customized HW accelerated kernels for the above operator sets , We can do away with the affine_map<(...)>  information also.

What does it need to be done in any reference sample or pointer would be appreciated?
 
   %2 =  linalg.init_tensor [2, 112, 112, 1, 64] : tensor<2x112x112x1x64xf32>
  %cst =  constant 0.000000e+00 : f32
  %3 =  linalg.fill(%2, %cst) : tensor<2x112x112x1x64xf32>, f32 -> tensor<2x112x112x1x64xf32>
  **%4 =  linalg.depthwise_conv_2d_input_nhwc_filter_hwcf {strides =  dense<2> : tensor<2xi64>} ins(%arg0, %arg1 : tensor<2x224x224x3xf32>, tensor<7x7x3x64xf32>) outs(%3 : tensor<2x112x112x1x64xf32>) -> tensor<2x112x112x1x64xf32>**
  %5 =  linalg.tensor_reshape %4 [affine_map<(d0, d1, d2, d3, d4) -> (d0)>, affine_map<(d0, d1, d2, d3, d4) -> (d1)>, affine_map<(d0, d1, d2, d3, d4) -> (d2)>, affine_map<(d0, d1, d2, d3, d4) -> (d3, d4)>] : tensor<2x112x112x1x64xf32> into tensor<2x112x112x64xf32>
  %6 =  shape.shape_of %5 : tensor<2x112x112x64xf32> -> tensor<?xindex>
  %7 =  shape.to_extent_tensor %6 : tensor<?xindex> -> tensor<4xindex>
  %8 =  linalg.init_tensor [2, 112, 112, 64] : tensor<2x112x112x64xf32>
  %9 =  linalg.generic {indexing_maps =  [affine_map<(d0, d1, d2, d3) -> (d3)>, affine_map<(d0, d1, d2, d3) -> (d0, d1, d2, d3)>], iterator_types =  ["parallel", "parallel", "parallel", "parallel"]} ins(%arg2 : tensor<64xf32>) outs(%8 : tensor<2x112x112x64xf32>) {
  ^bb0(%arg321: f32, %arg322: f32):  // no predecessors
    linalg.yield %arg321 : f32
  } -> tensor<2x112x112x64xf32>
  %10 =  linalg.init_tensor [2, 112, 112, 64] : tensor<2x112x112x64xf32>
  %11 =  linalg.generic {indexing_maps =  [affine_map<(d0, d1, d2, d3) -> (d0, d1, d2, d3)>, affine_map<(d0, d1, d2, d3) -> (d0, d1, d2, d3)>, affine_map<(d0, d1, d2, d3) -> (d0, d1, d2, d3)>], iterator_types =  ["parallel", "parallel", "parallel", "parallel"]} ins(%5, %9 : tensor<2x112x112x64xf32>, tensor<2x112x112x64xf32>) outs(%10 : tensor<2x112x112x64xf32>) {
  ^bb0(%arg321: f32, %arg322: f32, %arg323: f32):  // no predecessors
    **%58 =  addf %arg321, %arg322 : f32**
    linalg.yield %58 : f32
  } -> tensor<2x112x112x64xf32>

**Comment by Saduf2019** [comment]
@dinkdeep 

Kindly open a [stackoverflow](https://stackoverflow.com/questions/tagged/tensorflow) issue for this as it is not a bug or feature request, Please post this kind of support questions at Stackoverflow. There is a big community to support and learn from your questions.
Thanks!

**Comment by dinkdeep** [comment]
This behavior observed is something specific to MLIR and HloLegalizeToLinalgPass in TensorFlow .  A corresponding stackoverflow issue might not be of much help.

**Comment by tilakrayal** [comment]
Hi,

Thank you for opening this issue. Since this issue has been open for a long time, the code/debug information for this issue may not be relevant with the current state of the code base.

The Tensorflow team is constantly improving the framework by fixing bugs and adding new features. We suggest you try the latest TensorFlow [version](https://pypi.org/project/tensorflow/) with the latest compatible hardware configuration which could potentially resolve the issue. If you are still facing the issue, please create a new GitHub issue with your latest findings, with all the debugging information which could help us investigate.

Please follow the [release notes](https://github.com/tensorflow/tensorflow/blob/master/RELEASE.md) to stay up to date with the latest developments which are happening in the Tensorflow space.
